src/python/Publisher/PublisherMaster.py

```diff
@@ -216,49 +216,52 @@ def active_tasks(self, db):
         """
 
         self.logger.debug("Retrieving publications from oracleDB")
+        filesToPublish = []
         uri = self.REST_filetransfers
-        fileDoc = {}
-        fileDoc['asoworker'] = self.config.asoworker
-        fileDoc['subresource'] = 'acquirePublication'
-        data = encodeRequest(fileDoc)
-        results = ''
-        try:
-            results = db.post(uri=uri, data=data)
-        except Exception as ex:
-            self.logger.error("Failed to acquire publications from %s: %s", uri, ex)
-            return []
-
-        self.logger.debug("Retrieving max.100000 acquired publications from oracleDB")
-        fileDoc = dict()
-        fileDoc['asoworker'] = self.config.asoworker
-        fileDoc['subresource'] = 'acquiredPublication'
-        fileDoc['grouping'] = 0
-        fileDoc['limit'] = 100000
-        result = []
-        uri = self.REST_filetransfers
-        data = encodeRequest(fileDoc)
-
-        try:
-            results = db.get(uri=uri, data=data)
-            result.extend(oracleOutputMapping(results))
-        except Exception as ex:
-            self.logger.error("Failed to acquire publications from %s: %s", uri, ex)
-            return []
-
-        self.logger.debug("publen: %s", len(result))
+        asoworkers = self.config.asoworker
+        # asoworkers can be a string or a list of strings
+        # but if it is a string, do not turn it into a list of chars !
+        asoworkers = [asoworkers] if isinstance(asoworkers, basestring) else asoworkers
+        for asoworker in asoworkers:
+            self.logger.info("Processing publication requests for asoworker: %s", asoworker)
+            fileDoc = {}
+            fileDoc['asoworker'] = asoworker
+            fileDoc['subresource'] = 'acquirePublication'
+            data = encodeRequest(fileDoc)
+            try:
+                result = db.post(uri=uri, data=data)
```

Comment:
pylint complains about  `Unused variable 'result'`

Reply:
I know, but since that call returns something, if one has to go through with debugger it helps to have the return values assigend to something, even if there's no use for this `result` now. I I tried to think of some elegant alternative, but failed. Of course one can put some code on the tune of if result is bad print an error, but if calls fail we get an exception.

Reply:
thanks, should we silent it or simply leave it like it is? because other than this warning, there are no more major pylint warnings to be fixed

Reply:
I could not make my mind about which way to pick and handed to you. If you have no preference or ideas either, let's add a dummy line which references the unused variables. Something like a second call to logger in case of exception